spec/carrier_account_spec.rb

```diff
@@ -14,6 +14,16 @@
       # Remove the carrier account once we have tested it so we don't pollute the account with test accounts
       carrier_account.delete
     end
+
+    it 'sends FedexAccount to the correct endpoint' do
+      allow(EasyPost).to receive(:make_request).with(
+        :post, '/v2/carrier_accounts/register', nil, { carrier_account: { type: 'FedexAccount' } },
+      ).and_return({ 'id' => '123' })
```

Comment:
Returning the correct type if ID will tell the lib that this response is a CarrierAccount object.

```suggestion
      ).and_return({ 'id' => 'ca_123' })
```